websites/M/Mavanimes/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "Mavanimes",
+  "author": {
+    "name": "Adrien",
+    "id": "256458960666624003"
+  },
+  "category": "anime",
+  "logo": "http://www.mavanimes.co/wp-content/uploads/2018/03/cropped-mav-270x270.jpg",
```

Comment:
Logo must be on imgur

Reply:
Do it have to be on Imgur or it's ok like i did ?

Reply:
![image](https://user-images.githubusercontent.com/64903135/130376228-4762883b-c38e-4826-81b5-a548b5fb7587.png)

Reply:
Ok, I'm changing it

Reply:
Ok, I'm changing it